types/stripe-node/index.d.ts

```diff
@@ -4246,6 +4254,11 @@ declare namespace StripeNode {
              * The ID of the plan whose subscriptions will be retrieved
              */
             plan?: string;
+
+            /**
+             * The status of the subscriptions to retrieve.
+             */
+            status?: ISubscription["status"] | "all";
```

Comment:
`ISubscription["status"]` should have a type alias.

Reply:
Could you give me a nudge in the right direction? Do you mean something like changing `interface ISubscription { status: SubscriptionStatus }` and here `status?: SubscriptionStatus | "all"`?

Reply:
:+1:

Reply:
Done 👍